scripts/linux/buildDiagnostics.sh

```diff
@@ -5,6 +5,9 @@
 # It then publishes the binaries along with their corresponding dockerfiles to the publish directory,
 # so that buildImage.sh can build the container image.
 
+BuildConfiguration=${1:-release}
```

Comment:
In this script variables are in UPPERCASE_SNAKE format, not pascal.

Reply:
updated